Rakefile

```diff
@@ -15,8 +15,14 @@ require 'rake'
 require 'rspec/core'
 require 'rspec/core/rake_task'
 require 'rubocop/rake_task'
+
 RSpec::Core::RakeTask.new(:spec)
 
+RSpec::Core::RakeTask.new(:ascii_spec) do |t|
+  t.ruby_opts = '-E ISO-8859-1'
```

Comment:
The rake task name doesn't line up with the encoding - could the encoding be `ASCII` rather than Latin1?